sqlserver/datadog_checks/sqlserver/utils.py

```diff
@@ -137,3 +137,23 @@ def is_azure_sql_database(engine_edition):
     :return: bool
     """
     return engine_edition == ENGINE_EDITION_SQL_DATABASE
+
+
+def execute_query_output_result_as_dicts(query, cursor, convert_results_to_str=False, parameter=None):
```

Comment:
This name is a bit verbose, could call it `execute_query` and use an explicit return type

Reply:
sure, thx

Reply:
yeah missed this one, now done